components/multiselect/multiselect.tsx

```diff
@@ -360,76 +493,55 @@ export default class MultiSelect<T extends Value> extends React.PureComponent<Pr
-            <div className='filtered-user-list'>
-                <div className='filter-row filter-row--full'>
-                    <div className='multi-select__container react-select'>
-                        <ReactSelect
-                            id='selectItems'
-                            ref={this.reactSelectRef as React.RefObject<any>} // type of ref on @types/react-select is outdated
-                            isMulti={true}
-                            options={this.props.options}
-                            styles={styles}
-                            components={{
-                                Menu: nullComponent,
-                                IndicatorsContainer: nullComponent,
-                                MultiValueLabel: paddedComponent(this.props.valueRenderer),
-                            }}
-                            isClearable={false}
-                            openMenuOnFocus={false}
-                            menuIsOpen={false}
-                            onInputChange={this.onInput}
-                            onKeyDown={this.onInputKeyDown as React.KeyboardEventHandler}
-                            onChange={this.onChange}
-                            value={this.props.values}
-                            placeholder={this.props.placeholderText}
-                            inputValue={this.state.input}
-                            getOptionValue={(option: Value) => option.id}
-                            getOptionLabel={this.props.ariaLabelRenderer}
-                            aria-label={this.props.placeholderText}
-                            className={this.state.a11yActive ? 'multi-select__focused' : ''}
-                            classNamePrefix='react-select-auto react-select'
-                        />
-                        <SaveButton
-                            id='saveItems'
-                            saving={this.props.saving}
-                            disabled={this.props.saving}
-                            onClick={this.handleOnClick}
-                            defaultMessage={buttonSubmitText}
-                            savingMessage={this.props.buttonSubmitLoadingText}
-                        />
-                    </div>
-                    <div
-                        id='multiSelectHelpMemberInfo'
-                        className='multi-select__help'
-                    >
-                        {numRemainingText}
-                        {memberCount}
+            <React.Fragment>
+                <div className='filtered-user-list'>
+                    <div className='filter-row filter-row--full'>
+                        <div className='multi-select__container react-select'>
+                            {reactSelectDiv}
+                            {this.props.saveButtonPosition === 'top' &&
+                            <SaveButton
+                                id='saveItems'
+                                extraClasses='ml-4'
```

Comment:
is it possible to use classNames for the extra classes instead of having this prop?

Reply:
this comment ^

Reply:
Not sure how we do this.
Do I add a className prop to the component itself and then use classNames in it?

Furthermore, we are also moving the saveButton itself in the DOM based on the prop value, rather than just changing the class.

Reply:
Though I'm just removing the class altogether, and giving it through css.